dependency/vault_pki.go

```diff
@@ -124,34 +129,44 @@ func goodFor(cert *x509.Certificate) (time.Duration, bool) {
 		// between 87% and 93%
 		gooddur = gooddur + ((gooddur / 100) * int64(r.Intn(6)-3))
 	}
-	sleepFor := time.Duration(gooddur * 1e9)
+	sleepFor := time.Duration(gooddur * 1e9) // *1e9 converts nanos->seconds
```

Comment:
I'd say `seconds->nanos`, or just use `gooddur * time.Second`

Reply:
Oh.. you meant to just put the `goodur * time.Second` in the comment. 
Really missing that face-palm emoji here today.